### ISSUE TITLE
Add documentation for error responses when download fails in transportar

### ISSUE THREAD

**Issue by nuest** [o2r-project/o2r-web-api#46]
When there is no job and subsequently not image tarball that can be downloaded, then the transportar returns an HTTP `500` with a JSON body error message. This is not documented in the API, so please add it.

**Comment by LukasLohoff** [comment]
Is just the error message (similar to the documentation for other routes) enough?

```json
500 Internal Server Error

{"error":"no job found for this compendium, run a job before downloading with image"}
```

or do you think further explanation is necessary?

**Comment by nuest** [comment]
Yes, that's fine.

**Comment by LukasLohoff** [comment]
Fixed with #48
